test/lint/commit-script-check.sh

```diff
@@ -28,7 +62,11 @@ for commit in $(git rev-list --reverse $1); do
             echo "Error: missing script for: $commit"
             echo "Failed"
             RET=1
+        elif verify_script "$SCRIPT"; then
+            echo ERROR: You are using BSD sed syntax for "-i" option, which is incompatible with GNU sed
+            RET=1
         else
+            verify_script "$SCRIPT"
```

Comment:
Why run `verify_script` twice when it fails (in both the `elif` condition and `else` branch)?